pkg/k8s/object/builders/k8s-sensor/deployment/deployment.go

```diff
@@ -139,6 +139,8 @@ func (d *deploymentBuilder) build() *appsv1.Deployment {
 							Name:            "instana-agent",
 							Image:           d.Spec.K8sSensor.ImageSpec.Image(),
 							ImagePullPolicy: d.Spec.K8sSensor.ImageSpec.PullPolicy,
+							Command:         []string{"/ko-app/k8sensor"},
+							Args:            []string{"5s"},
```

Comment:
I think this needs the `-pollrate` before the `5s`

Reply:
Very true. Done.